packages/monosize/src/commands/measure.mts

```diff
@@ -15,10 +15,11 @@ import type { BuildResult } from '../types.mjs';
 export type MeasureOptions = CliOptions & {
   debug: boolean;
   'artifacts-location': string;
+  fixtures?: string;
```

Comment:
this needs to be added to `packages/monosize/src/commands/measure.mts` command `builder`